python/phase-2.md

```diff
@@ -0,0 +1,831 @@
+# Python SDK - Phase 2
+
+The Python SDK will be implemented in three loosely defined phases:
+
+* Phase 1 - Initial project, activity worker, and client
+* Phase 2 - Workflow worker, local activities, async activity completion, etc
+* Phase 3 - Deterministic workflow sandbox, replayer, test suite (some parts may be combined with phase 2)
+
+This proposal is for phase 2. It is intentionally loosely defined and all aspects are subject to discussion/change.
+
+## Overview
+
+Following up from phase 1 and known implementation changes since then, this phase will implement the actual workflow
+worker/code and other advanced features.
+
+A simple example of a workflow definition and worker run:
+
+```python
+from datetime import timedelta
+from temporalio import activity, workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+@activity.defn
+def say_hello(name: str) -> str:
```

Comment:
nit: doesn't this require registering an executor with the worker?

Reply:
It does, I forgot `async` on here, will fix.

Reply:
Fixed